fhir-persistence-schema/README.md

```diff
@@ -83,39 +110,24 @@ Edit `wlp/usr/servers/fhir-server/config/TNT1/fhir-server-config.json` and add t
 --tenant-key "<the-base64-tenant-key>"
 ```
 
-## Update the data (not admin) procedures only
+## Update the stored procedures for FHIRDATA (and not FHIR_ADMIN)
 
 ```
 --prop-file config/fhir.properties
 --schema-name FHIRDATA
 --update-proc
 ```
 
-# Printing the Schema
-
-To run this code, build the jar (fhir-database-utils and fhir-persistence-schema)
-
-``` shell 
-java -cp ./fhir-database-utils.jar:fhir-persistence-schema.jar com.ibm.fhir.schema.app.SchemaPrinter [--to-file]
-```
-
-Without to-file, the output is the current System.out else it's schema.sql, grants.sql and stored-procedures.sql of the current directory.
- 
-For db2 import to
-- schema.sql `db2 -tvf schema.sql`
-- grants.sql `db2 -tvf grants.sql`
-- stored-procedures.sql `db2 -td@ -vf stored-procedures.sql`
+# Alternative: manually apply the schema changes
 
-The schema is then available for use. 
+To manually apply the DDL to a Db2 instance:
+1. Print the schema to files by executing the SchemaPrinter:
+    `java -cp ./fhir-database-utils.jar:fhir-persistence-schema.jar com.ibm.fhir.schema.app.SchemaPrinter --to-file`
 
-# Creating the database 
+2. Connect to your instance and execute each of the following:
+    - schema.sql:  `db2 -tvf schema.sql`
+    - grants.sql:  `db2 -tvf grants.sql`
+    - stored-procedures.sql:  `db2 -td@ -vf stored-procedures.sql`
 
-To create the database and database user, please use the following command.
```

Comment:
please add this back.  I actually ran into an issue, and found this helpful (when setting up a local docker image).    Maybe add as an appendix?

Reply:
nevermind (didn't link with the above)

Reply:
yeah, i felt like "how to see what the schema Main app is doing" should go at the top, but still tried to be explicit that users can either:
* A. deploy the schema with the tool; or
* B. manually apply the DDL from the printed SQL files